test/cstrings/config.yml

```diff
@@ -0,0 +1,2 @@
+skip: false
```

Comment:
Not sure why this directory should be called `cstrings` rather than `string` or `strings`.